Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1462,51 +1468,139 @@ extension BrowserViewController: WKUIDelegate {
         return tab.webView
     }
 
+    func createDisableDialoguePrompt(alertController: UIAlertController) {
+        var toggle = UISwitch()
+        toggle.transform = CGAffineTransformMakeScale(0.75, 0.75)
+        toggle.addTarget(self, action: "disableDialogue:", forControlEvents: UIControlEvents.ValueChanged)
+
+        var disableDialogue = UITableViewCell()
+        disableDialogue.textLabel?.text = "Disable additional page dialogs"
+        disableDialogue.textLabel?.font = UIFont.systemFontOfSize(13)
+
+        var lineView = UIView(frame: CGRectMake(0, 5, disableDialogue.contentView.bounds.size.width, 0.3))
+        lineView.backgroundColor = UIColor.blackColor()
+        disableDialogue.frame = CGRectMake(-5, 70, 220, 50)
+        disableDialogue.contentView.addSubview(lineView)
+        disableDialogue.backgroundColor = UIColor.clearColor()
+
+        alertController.view.addSubview(disableDialogue)
+        alertController.view.addSubview(toggle)
+        alertController.view.clipsToBounds = true
```

Comment:
this is so the line doesn't extend past the alertController (setting the line to the width of the controller is apparently too big?? what)